docs/api.md

```diff
@@ -78,6 +78,7 @@
   * [page.select(selector, ...values)](#pageselectselector-values)
   * [page.setContent(html)](#pagesetcontenthtml)
   * [page.setCookie(...cookies)](#pagesetcookiecookies)
+  * [page.setDefaultNavigationTimeout(timeout)](#pagesetdefaultnavigationtimeouttimeout)
```

Comment:
So this setting affects 5 methods: `page.goto`, `page.goBack`, `page.goForward`, `page.reload` and `page.waitForNavigation`.
Can you please:
- refer to the methods from the doc of the method, explaining that this method will change defaults for them, and that the default value is 30 seconds
- update documentation for the methods themselves, citing `page.setDefaultNavigationTimeout`.

This should make it more understandable for our clients.